src/tests/functional/plugin/conformance/subgraphs_dumper/include/matchers/subgraph/fused_names.hpp

```diff
@@ -15,6 +15,8 @@ namespace subgraph_dumper {
 class FusedNamesExtractor : public SubgraphExtractor {
 public:
     FusedNamesExtractor();
+    ~FusedNamesExtractor();
```

Comment:
should both `SubgraphExtractor` and `FusedNamesExtractor` have virtual dtors?

Reply:
@ilya-lavrenov I wanted to check this mem leak only :)
the best decision is to mark extractors as `final` and use default destructors. As you can see, I added only one destructor for `FusedNames extractor` only to avoid of lost cache.